recipe/meta.yaml

```diff
@@ -41,3 +41,4 @@ about:
 extra:
   recipe-maintainers:
     - jtpio
+    - fcollonval
```

Comment:
cc @fcollonval if you agree to be listed there

Reply:
I agree 
Thanks @jtpio